src/globals/brand-colour.ts

```diff
@@ -1,14 +1,20 @@
+/**
+ * @see $brandColours in src/globals/variables.scss . This and that should be updated together.
+ *
+ * @todo consider changing values to remove need for comments. Not doing right now to avoid possible semantic
+ * merge conflicts.
+ */
 export type brandColour =
-  | 'primary'
-  | 'secondary'
-  | 'tertiary'
-  | 'brand-1'
-  | 'brand-2'
-  | 'brand-3'
-  | 'brand-4'
-  | 'brand-5'
-  | 'brand-6'
-  | 'black'
+  | 'primary' // blue
+  | 'secondary' // green
+  | 'tertiary' // coral
+  | 'brand-1' // cc-red
+  | 'brand-2' // wgmf-purple
+  | 'brand-3' // gmf-green
+  | 'brand-4' // emf-yellow
+  | 'brand-5' // c4c-orange
+  | 'brand-6' // mhf-turquoise
   | 'white'
+  | 'black'
   | 'grey-light'
   | 'grey-dark';
```

Comment:
should we add `grey-extra-light` and `grey-medium` here to match what we have in variables.scss ? Or are those colours that we've decided to stop using?

Reply:
Yes - we should add. Do you want me to add them?

Reply:
@tomwpaterson-shape No I've just pushed a commit here to add them.